app/views/shared/_asia_fields.html.slim

```diff
@@ -1,80 +1,70 @@
-                td 
-                  = f.input_field :preservation_motor_level_right_id,
-                  label: false,
-                  collection: Domain::LevelOfInjury.cached_all
-      .medium-6.columns
-        label ASIA Impairment Scale
-        .form-control.input Missing JS auto-calc
-        / TODO: when these are functional we won't need breaks
-        br
-        label
-          .checkbox-inline
-            = f.input :has_motor_or_sensory_asymmetry
-      .medium-6.columns
-        label Neurological Level of Injury
-        .form-control.input Missing JS auto-calc
-        br
-        / TODO: when these are functional we won't need breaks
-        = f.input :is_complete, 
-        collection: [["Complete", true], ["Incomplete", false] ],
-        label: false,
-        as: :radio_buttons, 
-        item_wrapper_class: 'radio-inline'
+      .medium-4.columns
+        = f.input :neurological_sensory_level_right_id,
+        collection: Domain::LevelOfInjury.cached_all,
+        label: "Sensory Right"
+        = f.input :neurological_sensory_level_left_id,
+        collection: Domain::LevelOfInjury.cached_all,
+        label: "Sensory Left"
+      .medium-4.columns.left
+        = f.input :neurological_motor_level_right_id,
+        collection: Domain::LevelOfInjury.cached_all,
+        label: "Motor Right"
+        = f.input :neurological_motor_level_left_id,
+        collection: Domain::LevelOfInjury.cached_all,
+        label: "Motor Left"
+  .row
+    .medium-4.columns
+      label ASIA Impairment Scale
+      / TODO: @awong add field for ASIA impairment Scale
+      = f.input :neurological_sensory_level_right_id,
+      label: false,
+      collection: Domain::LevelOfInjury.cached_all
+    .medium-4.columns.left
+      label Injury Complete or Incomplete
+      = f.input :is_complete, 
+      collection: [["Complete", true], ["Incomplete", false] ],
+      label: false,
+      as: :radio_buttons, 
+      item_wrapper_class: 'radio-inline'
+  .instrument#complete.hide
+    h4 If Complete, Fill in Information for Zone of Partial Preservation
+    .row
+      .medium-4.columns
+        = f.input :preservation_sensory_level_right_id,
+        collection: Domain::LevelOfInjury.cached_all,
+        label: "Sensory Right"
+        = f.input :preservation_sensory_level_left_id,
+        collection: Domain::LevelOfInjury.cached_all,
+        label: "Sensory Left"
+      .medium-4.columns.left
+        = f.input :preservation_motor_level_right_id,
+        collection: Domain::LevelOfInjury.cached_all,
+        label: "Motor Right"
+        = f.input :preservation_motor_level_left_id,
+        collection: Domain::LevelOfInjury.cached_all,
+        label: "Motor Left"
+
+javascript:
+  $(document).ready(function() {
+    showWhenValueEquals("[name='patient[asia_assessment][has_motor_or_sensory_asymmetry]']", "true", "#asymmetry");
+    showWhenValueEquals("[name='patient[asia_assessment][is_complete]']", "true", "#complete");
+  });
```

Comment:
@awong-dev I put js for partials inline. let me know if you have a better idea, but this seems to work fine for now. (other than that it throws off the unique identifiers to have it loading on the page twice.)